src/report_error.mli

```diff
@@ -29,6 +29,3 @@ val register : (exn -> printer option) -> unit
 
 (** Find an error printer *)
 val find_printer : exn -> printer option
-
-(**/**)
-val map_fname : (string -> string) ref
```

Comment:
This one is completely undocumented, but we are using it at Jane Street. We will be able to get rid of it  eventually once we switch from Jenga to Dune, but it will make our life easier if we keep it in the meantime.

FTR, this is what we use it for: internally, we have a really big mono-repository. We extract sub-directories from this mono-repo as separate projects that form the project we expose on github. While doing so, we use `dune external-lib-deps` to automatically generate the opam files. We use it as a library and use `map_fname` so that errors about files in the destination repositories are reported on the original files in our mono-repo and we can use the "jump to error" function of the editor. The reason we use it as a library is historical. We could change that, but given that we are planning to switch to dune, it doesn't seem worth changing this code now.

Reply:
OK! I'll just refactor that part a bit instead of removing it.